examples/tabs/palette/demo/@misk/core/ducks/simpleDucksUtilities.d.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * simple*Ducks libraries common code
+ */
+/**
+ * Handler Functions
```

Comment:
I feel like I already reviewed this code?

Reply:
Yes `examples/tabs/palette/demo/` is the build folder and includes `@misk/` libraries and other code necessary to run.